model/src/w3wavemd.F90

```diff
@@ -1762,27 +1763,48 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
 #endif
 
 #ifdef W3_MEMCHECK
-       write(740+IAPROC,*) 'memcheck_____:', 'WW3_WAVE TIME LOOP 13'
+       write(40000+IAPROC,*) 'memcheck_____:', 'WW3_WAVE TIME LOOP 13'
        call getMallocInfo(mallinfos)
-       call printMallInfo(IAPROC,mallInfos)
+       call printMallInfo(IAPROC+40000,mallInfos)
 #endif
 !
-        IF ( FLSOU .and. LPDLIB) THEN
-!
+#ifdef W3_PDLIB
+        IF ( FLSOU .and. LPDLIB .and. FSSOURCE) THEN
+#endif
+
+!!AR: This OpenMP statement is wrong
```

Comment:
There's an open issue https://github.com/NOAA-EMC/WW3/issues/639 so there's no need for this comment.

Reply:
Thanks, comment deleted.